_content/monographs/buddhist-cosmos_punnadhammo-mahathero.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "The Buddhist Cosmos: A Comprehensive Survey of the Early Buddhist Worldview; according to Theravāda and Sarvāstivāda sources"
+authors:
+  - "punnadhammo Mahathero"
+external_url: "https://arrowriver.ca/book/bookDown.php?format=pdf"
```

Comment:
The main page please: https://arrowriver.ca/book/cosmoBook.php

So people can choose the format they like